runeq/resources/project.py

```diff
@@ -110,13 +416,24 @@ def get_project(
     """
     client = client or global_graph_client()
     query = """
-        query getProject($project_id: ID) {
-            project (id: $project_id) {
+        query getProject($id: ID) {
+            project (id: $id) {
                 id,
                 title,
                 status,
                 description,
                 type,
+                cohortList {
+                    cohorts {
+                        id,
+                        title,
+                        description,
+                        created_at: createdAt,
+                        updated_at: updatedAt,
+                        created_by: createdBy,
+                        updated_by: updatedBy,
```

Comment:
just noticed - how come all the multi-word fields need the snakecase to camelcase conversion? 
does the literal query with just `createdAt` not work here?

Reply:
This is so it is easier to instantiate the results into each project! Since the attributes are camel case, a project can be created like this 
```
example_project = Project(**project_attrs)
```
instead of this
```
example_project = Project(
   created_at=project_attrs["createdAt"],
   updated_at=project_attrs["updatedAt],
   ...
)
```